src/CBT.NuGet/Tasks/AggregatePackages.cs

```diff
@@ -37,81 +48,7 @@ public sealed class AggregatePackages : Task
         [Required]
         public string PropsFile { get; set; }
```

Comment:
Are we guaranteed that PropsFile will always be a fully qualified path?  What is our assurance? Just because we are the only callers? Does it matter even if it isn't?